theHarvester/restfulHarvest.py

```diff
@@ -0,0 +1,46 @@
+import argparse
```

Comment:
git was not able to track this, but this was just copied from the top-level `restfulHarvest.py`. This was needed since the [`[project.scripts]`](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#entry-points) callable needs to be an importable from the distributed package (i.e. a file under `theHarvester/`)

Note I've kept the existing top-level `restfulHarvest.py` functional by just importing from here